model/task_lifecycle_test.go

```diff
@@ -5281,22 +5281,6 @@ func TestClearAndResetStrandedContainerTask(t *testing.T) {
 			assert.Equal(t, evergreen.CommandTypeSystem, dbTask.Details.Type)
 			assert.Equal(t, evergreen.TaskDescriptionStranded, dbTask.Details.Description)
 			assert.False(t, utility.IsZeroTime(dbTask.FinishTime))
-
-			// TODO (EVG-17033): if a stranded task hits the unschedulable
```

Comment:
This test still doesn't work, although EVG-17033 is closed. I don't know if this is worth us pursuing however.

Reply:
If EVG-17033 didn't get this to pass, then it probably didn't fix the bug. It should work ideally, but I don't particularly care, I only filed the ticket because I noticed it didn't work while I was testing something else.